libraries/TGRSIFormat/TChannel.cxx

```diff
@@ -401,8 +401,8 @@ double TChannel::CalibrateENG(double charge) {
    //integration parameter.
    if(ENGCoefficients.size()==0)
       return charge;
-   double cal_chg = 0.0;
-   for(int i=0;i<ENGCoefficients.size();i++){
```

Comment:
Fine. If we are looking for efficiency at that level, we may as well change our for loop to ++i instead of i++. 
i++ must allocate space to remember what i was before adding one while ++i does not. :P

> On Jul 13, 2015, at 2:49 PM, pcbend <notifications@github.com> wrote:
> 
> In libraries/TGRSIFormat/TChannel.cxx <https://github.com/GRIFFINCollaboration/GRSISort/pull/387#discussion_r34516230>:
> 
> > @@ -401,8 +401,8 @@ double TChannel::CalibrateENG(double charge) {
> >     //integration parameter.
> >     if(ENGCoefficients.size()==0)
> >        return charge;
> > -   double cal_chg = 0.0;
> > -   for(int i=0;i<ENGCoefficients.size();i++){
> The way Vinzenz changed it to does the same thing, just a bit more efficiently. var = vector.at(0) is slightly faster than variable =vector.at(0)* charge^0.
> … <x-msg://50/#>
> —
> Reply to this email directly or view it on GitHub.
> 
>  <https://github.com/GRIFFINCollaboration/GRSISort/pull/387#discussion_r34515499> <https://github.com/GRIFFINCollaboration/GRSISort/pull/387/files#r34515499>
> —
> Reply to this email directly or view it on GitHub <https://github.com/GRIFFINCollaboration/GRSISort/pull/387/files#r34516230>.
>